aci/types/linked_accounts.py

```
@@ -37,6 +37,23 @@
     enabled: bool | None = None
 
 
+class NoAuthSchemeCredentials(BaseModel):
+    """Limited no-auth credentials containing only access token"""
+
+    pass
+
+
+class OAuth2SchemeCredentialsLimited(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)